chapter_09_docker.asciidoc

```diff
@@ -496,7 +496,7 @@ Follow those, and then test your installation by running:
 
 [subs="specialcharacters,macros"]
 ----
-$ pass:quotes[*docker run busybox echo hello world*]
+$ pass:quotes[*docker run --rm busybox echo hello world*]
```

Comment:
i think i'm -1 on this one,  let's introduce the `--rm` flag later if we really want to. for now i want things as simple as possible

Reply:
I'll revert the edit to add `--rm`. 

Simple is good. And the reader can clean up later with 
```
docker image prune
docker container prune
```
```
docker rm CONTAINER
docker rmi IMAGE
```
or related commands.